pom.xml

```diff
@@ -82,7 +82,12 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-step-api</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>org.apache.sshd</groupId>
+      <artifactId>sshd-core</artifactId>
+      <version>2.5.1</version>
```

Comment:
Does not match PR summary. Which is right?

Reply:
I think the PR title is just misleading here. All you are doing is updating deps of this plugin so that it does _not_ use the `sshd` module/plugin at all, right?

Reply:
Yes, I changed the description but not the title, I just update the title